mesonbuild/backend/ninjabackend.py

```diff
@@ -2130,7 +2129,6 @@ def _link_library(libname: str, static: bool, bundle: bool = False):
             element.add_dep(deps)
         element.add_item('ARGS', args)
         element.add_item('targetdep', depfile)
-        element.add_item('cratetype', cratetype)
```

Comment:
Were we previously doing something with this ninja variable?

Reply:
Nope.